app/components/code-sample.js

```diff
@@ -1,14 +0,0 @@
-import Component from '@ember/component';
-import { registerHljsLanguage } from '../utils/beautify';
-
-export default Component.extend({
```

Comment:
It's not used, there is https://github.com/Wikia/design-system/blob/master/tests/dummy/app/components/code-sample.js